server/src/typeDefs/query.ts

```diff
@@ -3,8 +3,15 @@ import { gql } from 'apollo-server'
 export const query = gql`
   type Query {
     usersPagination(page: Float!, where: UserWhereInput): UsersPagination!
-    user(userId: String!): User!
 
+    user(userId: String!): User!
     me: User!
+
+    userFeeds: [Feed]
+		userArticles: [Article]
+		userSources: [Source]
+		feedSources(feedId: String!): [Source]
```

Comment:
so the thing about graphql is that you can query nested objects
https://www.digitalocean.com/community/tutorials/understanding-queries-in-graphql

so you want to do something like this:

```
type Query {
  userFeeds: [Feed]
}

type Feed {
   sources: [Source]
}

type Source {
  articles: [Article]
}
```

and you can retrieve all articles for a feed with something like this:
```
userFeeds {
   sources {
     articles {
        content
     }
   }
}
```